api/README.md

```diff
@@ -0,0 +1,57 @@
+# 🚀 Getting started with Strapi
```

Comment:
Can we update this README to ensure it feels like a README for this project rather than README of Strapi?

For example - it says:

```
npm run develop
# or
yarn develop
```

What is this boilerplate recommends to use - npm or yarn?

Similarly other places here.

README is a super important file for any new developer so they can get started easily. Given that this is a boilerplate and can be used across multiple pages, it becomes super important to write is accurately and beautifully.